code/game/gamemodes/intercept_report.dm

```diff
@@ -4,54 +4,54 @@
 	var/text
 
 /datum/intercept_text/proc/build(mode_type)
-	text = "<hr>"
+	text = ""
 	switch(mode_type)
 		if("blob")
-			text += "A CMP scientist by the name of [pick("Griff", "Pasteur", "Chamberland", "Buist", "Rivers", "Stanley")] boasted about his corporation's \"finest creation\" - a macrobiological \
+			text = "<hr>A CMP scientist by the name of [pick("Griff", "Pasteur", "Chamberland", "Buist", "Rivers", "Stanley")] boasted about his corporation's \"finest creation\" - a macrobiological \

```

Comment:
I mean, you could do this without putting a `<hr>` before each and every report.

Reply:
Should the `<hr>` be ditched entirely then?  If not, where should it go?

Reply:
It should not be ditched; it should appear before the text the switch produces if, and only if, the switch produces text. you can do this without putting it before each block of text.